javascript/index.html

```diff
@@ -111,7 +111,11 @@ <h4>
             <li>
                 <a href="./answers/q5.html"><h4>What are escape Sequences?</h4></a>
             </li>
-
+            <li>
+                <a href="./answers/var_vs_let.html">
```

Comment:
change the name of ans html file as the format done by others check that eg:{name}-q{no.}

Reply:
done.